pkg/kubernetes/scripts/details.js

```diff
@@ -434,9 +434,12 @@
         "KubeTranslate",
         function($q, $scope, $instance, dialogData, methods, translate) {
             var _ = translate.gettext;
-            var fields = {};
+            var item = dialogData.item;
+            var fields = {
```

Comment:
Should this come after validItem has had a chance to reject it. That way we can be sure item isn't null.